middlewares/lti.js

```diff
@@ -208,21 +208,23 @@ const serveDeliverableAsWord = (req, res) => {
 
       // remove HTML tags from user responses
       subDeliverable.activitiesArray.forEach(activity => {
-        activity.data = activity.data && activity.data.replace(/(<([^>]+)>)/ig, '')
-      })
-    })
+        activity.data = activity.data && activity.data.replace(/(<([^>]+)>)/ig, '');
+      });
+    });
 
     res.setHeader('Content-Type', 'application/vnd.openxmlformats-officedocument.wordprocessingml.document');
     res.setHeader('Content-Disposition', `attachment; filename=${type}-${email}.docx`);
-    res.send(wordExport.render(deliverable));
+
+    if (type === 'businessmodelcanvas') {
```

Comment:
Should it be better to have these type constants somewhere in the same file and import it when needed? I don't think it's a good idea using magic strings